src/verification/ScratchUserCheck.php

```diff
@@ -8,9 +8,9 @@ class ScratchUserCheck {
 
 	private static function fetchProfile($username, &$isScratcher, &$joinedAt, &$error) {
 		$url = sprintf(self::PROFILE_URL, $username);
-		$html = @file_get_contents($url, false, stream_context_create(
-			"http" => ["header" => "Accept-Language: en\r\nCookie: scratchlanguage=en"]
-		));
+		$html = @file_get_contents($url, false, stream_context_create([
+			'http' => ['header' => 'Accept-Language: en\r\nCookie: scratchlanguage=en']
```

Comment:
I suppose I should have been clearer with the single quotes: it's right for http and header, but for the body you actually do want double quotes due to the use of \r and \n.